debian/control

```diff
@@ -32,6 +32,7 @@ Build-Depends: debhelper (>= 10),
                libraft-dev (= 0.10.1-1409.gc354cd7),
                python3-tabulate,
                liblz4-dev,
+               libaio-dev,
```

Comment:
Also need a debian/changelog entry

Reply:
Thank you!  I will add it later once current CI tests pass and get more feedback.

Reply:
debian/changelog entry was added.